src/server/index.jsx

```diff
@@ -246,6 +485,10 @@ server
       });
     },
   )
+  .get('/:service/testdata.json', async ({ params }, res) => {
+    res.set('Cache-Control', 'public, max-age=31557600'); // testing the app respects the cache
```

Comment:
Does the app respect the cache? Do you mean whether serverside fetches automatically respect cache headers?

Reply:
This was to test that client side renders i.e. linking to different STY's and back (within the same service) respect the cache and they do. Once this data has been fetch once moving onto another STY in the same service the data will be fetched from browser cache.

Reply:
SSR caching I think is a whole different ball game and will require a cache on the server

Reply:
Got ya, because this component runs on both the client and the server, I'd assume then on the server the cache header is not respected and that is fine for a first implementation.